### PR TITLE
Confirmation for password

### PR DESCRIPTION
**Pull request by jp-justine**

### CODE REVIEW

File: src/Form/RegistrationFormType.php [hunk 1]
```diff
@@ -18,11 +19,14 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->add('email')
-            ->add('plainPassword', PasswordType::class, [
+            ->add('Password', RepeatedType::class, [
```
Comment (rocambille) [inline src/Form/RegistrationFormType.php]: Password => repeatedPassword

File: templates/registration/register.html.twig [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
     {{ form_start(registrationForm) }}
         {{ form_row(registrationForm.email) }}
-        {{ form_row(registrationForm.plainPassword, {
+        {{ form_row(registrationForm.Password, {
```
Comment (rocambille) [inline templates/registration/register.html.twig]: Password => repeatedPassword

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Form/RegistrationFormType.php [hunk 3]
```diff
@@ -19,7 +19,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
             ->add('email')
-            ->add('Password', RepeatedType::class, [
+            ->add('RepeatedPassword', RepeatedType::class, [
```
Comment (MarjorieThomassin) [inline src/Form/RegistrationFormType.php]: RepeatedPassword => repeatedPassword

File: templates/registration/register.html.twig [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
     {{ form_start(registrationForm) }}
         {{ form_row(registrationForm.email) }}
-        {{ form_row(registrationForm.Password, {
+        {{ form_row(registrationForm.RepeatedPassword, {
```
Comment (MarjorieThomassin) [inline templates/registration/register.html.twig]: RepeatedPassword => repeatedPassword

**Pull request merged**